File: MediaLibrary/MediaLibraryWebUI/Models/Configurations/TelevisionConfiguration.cs
using MediaLibraryWebUI.Models.Interfaces;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using static MediaLibraryWebUI.Enums;

namespace MediaLibraryWebUI.Models.Configurations
{
    public class TelevisionConfiguration : BaseConfiguration
    {
        public TelevisionConfiguration()
        {
            SelectedSeason = 1;
        }

        public int SelectedSeriesId { get; set; }
        public int SelectedSeason { get; set; }
        public TelevisionPages SelectedTelevisionPage { get; set; }
        public SeriesSort SelectedSeriesSort { get; set; }
    }
}

File: MediaLibrary/MediaLibraryWebUI/lib/app/configurations/television-configuration.ts
import BaseConfiguration from './base-configuration'
import * as Enums from '../assets/enums'

export default class TelevisionConfiguration extends BaseConfiguration {
    private selectedSeriesId: number;
    private selectedSeason: number;
    private selectedTelevisionPage: Enums.TelevisionPages;
    private selectedSeriesSort: Enums.SeriesSort;

    constructor() {
        super();
    }
}
